pkg/k8s/client_test.go

```diff
@@ -26,17 +27,11 @@ import (
 
 const testFieldManager = "eventing-manager"
 
+var errPatchNotAllowed = errors.New("apply patches are not supported in the fake client")
+
 func Test_PatchApply(t *testing.T) {
```

Comment:
I think, we should get rid of the test for PatchApply at all as we test it as a part of the integration tests. What do you think?

Reply:
I have no preference. We can keep it as having a test with simple checks is also a good idea.

Reply:
nitpick: please delete it. Little bit less test code :)

Reply:
done